### PR TITLE
Patron2020 359 extra request /embedded

### PR DESCRIPTION
**Pull request by kuba8686**

### CODE REVIEW

File: gateway/gateway.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#define CROW_ENABLE_SSL
+//#define CROW_ENABLE_SSL
```
Comment (szymoneiro) [inline gateway/gateway.cpp:1]: > Commented code should not be committed to repository.

File: gateway/gateway.cpp [hunk 2]
```diff
@@ -203,8 +219,8 @@ int main(void)
         return crow::response(200);
     });
 
-    app.port(443)
-    .ssl_file(CERT_FILE, KEY_FILE)
+    app.port(15443)
+    //.ssl_file(CERT_FILE, KEY_FILE)
```
Comment (szymoneiro) [inline gateway/gateway.cpp:223]: Again it's commented code.

File: gateway/sensors.h [hunk 3]
```diff
@@ -305,3 +320,10 @@ void from_json(const json &j, Keyboard &keyboard)
     //j.at("type").get_to(keyboard.type);
     j.at("message").get_to(keyboard.message);
 }
+
+void from_json(const json &j, MotionSensor &motion)
+{
+    j.at("id").get_to(motion.id);
+    //j.at("type").get_to(motion.type);
```
Comment (szymoneiro) [inline gateway/sensors.h:327]: Same thing

File: gateway/gateway.cpp (hunk 1 above)
Comment (kuba8686) [reply gateway/gateway.cpp]: fixed

File: gateway/dashboard.h [hunk 4]
```diff
@@ -135,14 +209,11 @@ class Dashboard
                     sensorExist = 1;
                     return 0;
                 }
-                if (sensorExist == 0) 
-                {
-                    add_window_blind(request);
-                    return 0;
-                }
+
```
Comment (Siwy160) [inline gateway/dashboard.h:212]: blank lines

File: gateway/dashboard.h [hunk 5]
```diff
@@ -135,14 +209,11 @@ class Dashboard
...
-                if (sensorExist == 0) 
-                {
-                    add_window_blind(request);
-                    return 0;
-                }
+                
             }
             
             
+
```
Comment (Siwy160) [inline gateway/dashboard.h:216]: 3 blank lines.

File: gateway/dashboard.h [hunk 6]
```diff
@@ -120,6 +150,50 @@ class Dashboard
...
+                    sensorExist = 1;
+                    return 0;
+                } 
+            }
+            if (sensorExist == 0)
+                {
+                    add_motion_sensor(request);
+                    return 0;
+                }
+
```
Comment (Siwy160) [inline gateway/dashboard.h:193]: blank line ;)

File: gateway/gateway.cpp (hunk 2 above)
Comment (kuba8686) [reply gateway/gateway.cpp]: fixed

File: gateway/sensors.h (hunk 3 above)
Comment (kuba8686) [reply gateway/sensors.h]: fixed

File: gateway/dashboard.h (hunk 4 above)
Comment (kuba8686) [reply gateway/dashboard.h]: fixed

File: gateway/dashboard.h (hunk 5 above)
Comment (kuba8686) [reply gateway/dashboard.h]: fixed

File: gateway/dashboard.h (hunk 6 above)
Comment (kuba8686) [reply gateway/dashboard.h]: fixed

**Pull request merged**